manifests/globals.pp

```diff
@@ -1,29 +1,118 @@
-# @summary Class for setting cross-class global overrides. See README.md for more details.
+# @summary Class for setting cross-class global overrides.
+#
+# @example  Use a more recent MongoDB version to install from the community repository.
+#
+#   class {'mongodb::globals':
+#     manage_package_repo => true,
+#     version             => '3.6',
```

Comment:
I don't think setting version to major.minor is working.
It will try to install packages with this version and they don't exist.

Change to repo_version?